miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -280,6 +280,145 @@
       <td>Emptied:</td>
       <td><form:checkbox id="empty" path="empty"/></td>
     </tr>
+    
+    <c:if test="${!empty sample.sampleAdditionalInfo}">
+            <div class="bs-callout bs-callout-info">
+              <p>Some sample fields shown below cannot be updated on this page. Use the bulk edit form to modify these fields.</p>
+            </div>
```

Comment:
Can we add a link to this page? 
`<a href='<c:url value="/miso/sample/bulk/edit/${sample.id}"/>/>Use the bulk edit form</a>`